OoyalaSkinSampleApp/OoyalaSkinSampleApp/Players/DoublePlayerPlayerViewController.m

```diff
@@ -0,0 +1,85 @@
+#import "DoublePlayerPlayerViewController.h"
+#import <OoyalaSkinSDK/OoyalaSkinSDK.h>
+#import <OoyalaSDK/OoyalaSDK.h>
+
+@interface DoublePlayerPlayerViewController ()
+
+@property (nonatomic, retain) OOSkinViewController *skinController;
```

Comment:
are you using this one? It seems like you instantiate OOSkinViewController and don't store a reference to it when creating the player